src/utilities/style-to-px.converter.ts

```diff
@@ -0,0 +1,26 @@
+export const styleToPixelConverter = (styleValue: string): number | null => {
+  const match = styleValue.match(/^(\d+(\.\d+)?)(.*)$/);
+
+  if (!match) return null;
+
+  const value = parseFloat(match[1]);
+  const unit = match[3];
+
+  let styleInPixel: number | null;
+
+  switch (unit) {
+    case "px":
+      styleInPixel = value;
+      break;
+    case "vh":
+      styleInPixel = (value * window.innerHeight) / 100;
+      break;
```

Comment:
Instead of using "vh", I suggest supporting "vw" because it represents a width attribute

Reply:
i fixed it, thanks